source/operations/checklists/hardware.rst

```diff
@@ -46,23 +49,38 @@ While MinIO can run on commodity or "budget" hardware, we strongly recommend usi
    * - :octicon:`circle`
      - Sufficient CPU cores to achieve performance goals for hashing (for example, for healing) and encryption
        
-       MinIO recommends Dual Intel® Xeon® Scalable Gold CPUs (minimum 16 cores per socket) or any CPU with AVX512 instructions
+       MinIO recommends Single Socket Intel® Xeon® Scalable Gold CPUs (minimum 16 cores per socket)
 
    * - :octicon:`circle`
      - Sufficient RAM to achieve performance goals based on the number of drives and anticipated concurrent requests (see the :ref:`formula and reference table <minio-hardware-checklist-memory>`).
 
        MinIO recommends a minimum of 128GB of memory per node for best performance.
 
    * - :octicon:`circle`
-     - Minimum of four nodes dedicated to object storage.
+     - .. cond:: k8s
+
+         MinIO requires a *minimum* of 4 worker nodes per MinIO Tenant
+
+         MinIO strongly recommends four distinct worker nodes dedicated to servicing the MinIO Tenant.
+         Colocating multiple high-performance softwares on the same nodes can result in resource contention and reduced overall performance for all softwares.
```

Comment:
`softwares` sounds odd to me, maybe `applications` or `services`?

Reply:
services!